en/mozorg/home-new.ftl

```diff
@@ -23,20 +23,46 @@ home-product-firefox-browsing = Private & secure browsing
 home-cta-get-firefox = Get { -brand-name-firefox }
 home-product-pocket-articles = The web’s most intriguing articles
 home-cta-get-pocket = Get { -brand-name-pocket}
+
+# Obsolete string (expires: 2024-07-16)
 home-product-relay-masks = Easy-to-use email & phone masks
+
+home-product-relay-hide-your = Hide your phone number and email from spammers
 home-cta-get-relay = Get { -brand-name-relay }
 home-product-vpn-trust = A VPN you can trust
 home-cta-get-vpn = Get { -brand-name-mozilla-vpn }
+
+# Obsolete string (expires 2024-07-23)
 home-product-monitor-data = Data breach alerts
+
+home-product-monitor-protect-your = Protectr your private info from data brokers
```

Comment:
**"Protectr"** is supposed to be "Protect"?

> ![Screen Shot 2024-05-25 at 14 52 36](https://github.com/mozilla-l10n/www-l10n/assets/1784648/d0f21ec7-ccf2-4d51-992f-7e621521a9ec)

Reply:
(Given the string id says "`protect-your`" I assume that it was indeed intended to be "Protect".)

I've opened https://github.com/mozilla/bedrock/pull/14602 for this, incl. the random whitespace.

Reply:
https://github.com/mozilla/bedrock/pull/14602 is merged :)